security/tpm/test/_tpm_test.js

```diff
@@ -116,13 +202,117 @@ describe('tpm', function () {
 
     it('correctly deals with error during tpm connection', function(done) {
       /*Tests_SRS_NODE_TPM_SECURITY_CLIENT_06_013: [If `signWithIdentity` is invoked without a previous successful invocation of `activateIdentityKey`, the callback will be invoked with `err` of `InvalidOperationError`.] */
-      var client = new TpmSecurityClient('registration', fakeSimpleTpmClient);
+      var tpm = new tss.Tpm();
+      var client = new TpmSecurityClient('registration', tpm);
       client.signWithIdentity([1], (err, signResult) => {
         assert.isOk(err, 'should be an error from the connection portion of starting the tpm');
```

Comment:
>assert.isOk(err, 'should be an error from the connection portion of starting the tpm'); [](start = 8, length = 87)

check the type? also not sure what "the connection portion of starting the tpm" is or means

Reply:
I really didn't want to depend whatever was returned from the .Connection call.

---
In reply to: [167386046](https://github.com/Azure/azure-iot-sdk-node/pull/225#discussion_r167386046) [](ancestors = 167386046)